content/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review.md

```diff
@@ -16,9 +16,11 @@ topics:
   - Pull requests
 shortTitle: Request a PR review
 ---
-Owners and collaborators on a repository owned by a user account can assign pull request reviews. Organization members with triage permissions to a repository can assign a pull request review.
+Repositories belong to a user account (a single individual owner) or an organization account (a shared account with numerous collaborators or maintainers). You can read more about the different account types [here](/get-started/learning-about-github/types-of-github-accounts). Owners and collaborators on a repository owned by a user account can assign pull request reviews. Likewise, organization members with triage permissions can also assign a pull request. 
 
-Owners or collaborators can assign a pull request review to any person that has been explicitly granted [read access](/articles/access-permissions-on-github) to a user-owned repository. Organization members can assign a pull request review to any person or team with read access to a repository. The requested reviewer or team will receive a notification that you asked them to review the pull request. {% ifversion fpt or ghae or ghes or ghec %}If you request a review from a team and code review assignment is enabled, specific members will be requested and the team will be removed as a reviewer. For more information, see "[Managing code review settings for your team](/organizations/organizing-members-into-teams/managing-code-review-settings-for-your-team)."{% endif %}
+In order to request a pull request review, you will need [write access](/organizations/managing-access-to-your-organizations-repositories/repository-roles-for-an-organization) permissions. Conversely, owners or collaborators can assign a pull request review to any person that has been granted [read access](/get-started/quickstart/github-glossary#read-access) only.
```

Comment:
EFL readers are sometimes confused by "In order", as it can suggest that ordering is involved. I'd drop this. 

Again, I'd also use a "For more information..." link here.

"Conversely" normally reverses the previous statement, which I don't think it does here. The mention of "owners or collaborators" (from the original) is a little misleading here, I think, because only collaborators with write access can assign reviewers. It's the write access (which you've added in the first sentence) that's the important thing. I'd reword this a little to clarify this.

```suggestion
To assign a reviewer to a pull request, you will need write access to the repository. For more information about repository access, see "[Repository roles for an organization](/organizations/managing-access-to-your-organizations-repositories/repository-roles-for-an-organization)." If you have write access, you can assign anyone who has read access to the repository as a reviewer.
```

Reply:
Again, thank you for the clarification and the tip on the EFL reader.